web/api/v6/report_server.thrift

```diff
@@ -603,4 +607,10 @@ service codeCheckerDBAccess {
   bool storeAnalysisStatistics(1: string runName
                                2: string zipfile)
                                throws (1: shared.RequestFailed requestError),
+
+  // Get analysis statistics for a run.
+  // PERMISSION: PRODUCT_STORE
```

Comment:
Product access right is not enough? At the api implementation we check for access rights.